### PR TITLE
Updating the setup steps in README - adding a distinct deps.get; Addi…

### PR DESCRIPTION
**Pull request by marjaimate**
…ng minimum requirements for DB.

### CODE REVIEW

File: test/support/dummy/config/config.exs [hunk 1]
```diff
@@ -22,7 +22,8 @@ config :dummy, Dummy.Repo,
   adapter: Ecto.Adapters.Postgres,
   pool: Ecto.Adapters.SQL.Sandbox,
   database: "ryal_dummy_#{Mix.env}",
-  username: System.get_env("DUMMY_DB_USER") || System.get_env("USER")
+  username: System.get_env("DUMMY_DB_USER") || "postgres",
+  password: "postgres"
```
Comment (BenMorganIO) [inline test/support/dummy/config/config.exs]: This fails for my PG database. I'm thinking this should be moved to an environment variable that a person can configure.

Comment (marjaimate) [inline test/support/dummy/config/config.exs]: Sure thing, I'll correct it

Comment (BenMorganIO) [inline test/support/dummy/config/config.exs]: I wonder if there's something like a `dot-env` for elixir...

Comment (yordis) [inline test/support/dummy/config/config.exs]: @BenMorganIO the Elixir way is to use `Config`. If you dont want to have this issue all over the place just add `import_config "config.secret.exs"` or which ever convention where users can override the configuration. So I would push the database connection there.

Comment (BenMorganIO) [inline test/support/dummy/config/config.exs]: I've merged this, I'm just going to go in and setup the `config.secret.exs` though in another branch :p

**Pull request merged**